snapdenv/useragent.go

```diff
@@ -44,6 +44,9 @@ func SetUserAgentFromVersion(version string, probeForceDevMode func() bool, extr
 	if Testing() {
 		extras = append(extras, "testing")
 	}
+	if release.OnWSL {
+		extras = append(extras, "wsl")
+	}
```

Comment:
nitpick: I have a slight preference for swapping things such that testing still comes last when present

Reply:
I'm sorry, I don't quite understand what you are saying. What do you think should be swapped?